doc/doxygen/headers/iterators.h

```diff
@@ -383,7 +383,7 @@ levels, and accessors for objects of lower dimensionality do not have a
 @ingroup grid
 */
 
-//@}
+/** @} */
```

Comment:
To me, this looks not as nice as would
```suggestion
/// @}
```
As far as I know, doxygen should also pick up `///` comments. If others prefer the version here, I am happy to support that as well.

Reply:
Yes, those work as well.

Reply:
My motivation was making it symmetric with the opening @{ which can be grouped with @name etc.